pkg/provision/workspace/ssh-askpass.sh

```
@@ -1,4 +1,8 @@
 #!/bin/sh
+# Only handle passphrase prompts - exit silently for other prompts (e.g., HTTPS git repo username/password request)
+if [[ "$1" != "Enter passphrase for key '"* ]]; then
```

Shellcheck (0.11.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)